TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -365,5 +365,74 @@ public LockTableStatement CreateLockTableStatement(CodeElementsParser.LockTableS
 
             return new LockTableStatement(tableName, partitionId, mode);
         }
+
+        public AlterSequenceStatement CreateAlterSequenceStatement(
+            CodeElementsParser.AlterSequenceStatementContext context)
+        {
+            SymbolReference sequenceName = null;
+            SyntaxProperty<bool> restart = null;
+            SqlConstant restartValue = null;
+            SqlConstant incrementValue = null;
+            SqlConstant minValue = null;
+            SqlConstant maxValue = null;
+            SyntaxProperty<bool> cycle = null;
+            SqlConstant cacheSize = null;
+            SyntaxProperty<bool> ordered = null;
+            if (context.sequence_name != null)
+            {
+                sequenceName = new SymbolReference(new AlphanumericValue((Token) context.sequence_name),
+                    SymbolType.SqlIdentifier);
+            }
+
+            foreach (var alterSequenceClause in context.alterSequenceClause())
```

Comment:
I see now what you mean about checking the non-duplicate clauses in the builder. I think we should test that for each clause the associated SqlConstant or SyntaxProperty is not null before defining it so as to keep only the first clause and identify the duplicates. We could then pass the duplicates to the checker so it can create the diagnostic.